src/css/index.css

```diff
@@ -171,3 +170,158 @@ header {
   top: 24px;
   left: 30px;
 }
+
+
+.header-down-arrow {
+  position: absolute;
+  width: 16px;
+  height: 6.67px;
+  left: 712px;
+  top: 723px;
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #SECTION FEATURE
+\* ---------------------------------------------------------------------- */
+
+
+.features {
+  position: absolute;
+  width: 1440px;
+  height: 768px;
+  left: 0px;
+  top: 768px;
+  background: var(--bg-color);
+}
+
+
+.features-title {
+  text-align: center;
+  margin: 80px 526px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: normal;
+  color: var(--txt-color-cstom);
+}
+
+
+.features-title-name {
+  font-size: 44px;
+  margin: 0 100px;
+}
+
+
+.features-title-subtitle {
+  font-size: 18px;
+  margin: 32px 0 0 0;
+  text-transform: uppercase;
+}
+
+
+.features-content {
+  margin: 32px 72px 59px 162px;
+  display: flex;
+}
+
+
+.features-content-left {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+
+.responsive-layout, 
+.pixel-perfect, 
+.organized-layers {
+  display: grid;
+  grid-template-columns: 60px 262px;
+}
+
+
+.responsive-layout-content,
+.pixel-perfect-content,
+.organized-layers-content {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.responsive-layout-content-title,
+.pixel-perfect-content-title,
+.organized-layers-content-title {
+  margin: 0 10px;
+  text-transform: uppercase;
+}
+
+
+.responsive-layout-content-description,
+.pixel-perfect-content-description,
+.organized-layers-content-description {
+  margin: 10px 10px 30px 10px;
+}
+
+
+.features-content-right {
+  background: url(../images/features/mac.png);
+  width: 802px;
+  height: 464px;
+  margin-left: 100px;
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #SECTION DOWNLOAD
+\* ---------------------------------------------------------------------- */
+
+
+.download {
+  position: absolute;
+  width: 1440px;
+  height: 680px;
+  left: 0;
+  top: 1536px;
+  background: linear-gradient(0deg, #4b4b4b, #4b4b4b), url(../images/download/background.png);
+  background-blend-mode: soft-light, normal;
+}
+
+
+.download-content {
+  text-align: center;
+  color: var(--txt-color-white);
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: normal;
+  margin-top: 163px;
+  margin-bottom: 45px;
+}
+
+
+.download-content-title {
+  font-size: 44px;
+  line-height: 48px;
+  margin-bottom: 20px;
+}
+
+
+.download-content-description {
+  width: 555px;
+  margin: 0 auto;
+}
+
+
+.download-button {
+  width: 210px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  background: #EBCB80;
+  margin: 0 auto;
+  border-radius: 50px;
+}
+
+
+.download-button a {
+  text-transform: uppercase;
```

Comment:
I see you use this style multiple time, you should define class for this style to reuse. Ex: .text-uppercase {} --> className="... text-uppercase"